src/elements/content-sidebar/versions/VersionsItemActions.js

```diff
@@ -101,13 +101,13 @@ const VersionsItemActions = ({
 
                 {showDownload && (
                     <VersionsItemAction action="download" fileId={fileId} isCurrent={isCurrent} onClick={onDownload}>
-                        <IconDownload {...ICON_SIZE} />
+                        <IconDownload />
```

Comment:
It looks like the new icons are larger than they were before, so I think the set size would still be needed to maintain consistency. Can you post a before/after screenshot of this dropdown?

Reply:
Zeplin for menu styling: zpl.io/bWjPl4P

Icon size 16x16, color #4e4e4e (bdlGray80)

Before: 
![image](https://user-images.githubusercontent.com/5782586/67976900-3d347f80-fbd4-11e9-981b-44feb99fda0d.png)

After: 
![image](https://user-images.githubusercontent.com/5782586/67976702-d9aa5200-fbd3-11e9-9534-a3f76131d48f.png)